NEWS.md

```diff
@@ -4,6 +4,8 @@
 
 #### NEW FEATURES
 
+1. `:=` no longer recycles too-short RHS vectors to match the LHS length. There was a warning when recycling left a remainder but no warning when the LHS length was an exact multiple of the RHS length (the same behaviour as base R). All recycling is now an error unless `length(RHS)==1` or `length(RHS)==length(LHS)`. Consistent feedback was that recycling is rarely useful and more often a bug. In rare cases where you need to recycle a length>1 vector, use `rep()` explicitly. Single values are still recycled silently as before.
```

Comment:
might be worth doing a revdep run with this commit, i'm curious how many dependencies assume this behavior

Reply:
Running. First 80 of 750 complete. Just 1 affected so far: antaresViz

Reply:
Complete. 16 affected out of 748.  I've updated the top comment.